index.rst

```diff
@@ -48,7 +48,353 @@
+It is therefore necessary to understand what Rubin Science Platform support will be provided for these data products at that time, and to develop and exercise the associated data release procedures now, prior to the beginning of ComCam and LSSTCam commissioning.
+There are planned capabilities of the operations-era system with some complexity, such as the management of a 30-day cache of uncompressed (or losslessly compressed) PVIs and difference images from AP, and (possibly) the replacement of the PVIs with lossy-compressed versions.
+We have to devise a system that works in the context of the transfer, around the 80-hour mark, of images from the "embargo" systems to the externally visible ones.
+We also have to manage in a clean way the communication to users of the fact that, during early operations,
+some observations will have templates available, and therefore yield difference images and DIA catalog data, while others will not.
+We also must communicate the progressive increase in the proportion of sky covered by templates.
+
+We would also like the system to meet some basic user-communication goals such as "where did LSSTCam look last night?", and the like.
+
+The AuxTel LATISS imaging surveys provide a useful integration target for continuous release of data products produced by Alert Production.
+Prompt Processing payloads are now running image processing pipelines on LATISS data in real-time, replicating the interfaces that will exist with LSSTCam at lower data volumes.
+
+At this time there is no plan to release commissioning-era AuxTel imaging data or alerts publicly to science users or to alert brokers, so access to these products will be restricted to project members and SITCOM in-kind contributors.
+
+
+Schedule
+========
+
+We are planning toward the following objectives:
+
+AP operations
+-------------
+
+* First day running AP against Cassandra is attempted - mid-February 2024
+* Successfully running AP against Cassandra on AuxTel data - mid-March 2024
+* First day to attempt replicating Cassandra to Postgres - beginning of March 2024
+* Stable running of AP on Cassandra on AuxTel data with replication to Postgres - end of March 2024
+
+There is comfortable schedule margin available for these activities up through the start of May
+if there are unexpected problems.
+
+* Implementation of automated post-embargo sweeps to ``/repo/prompt`` - (TBD)
+* Implementation of 30-day cache purges - TBC if this will even be done in early operations
+  
+
+Data services
+-------------
+
+* Exposure of ``/repo/embargo`` images via image services - prototype currently running
+* Exposure of the expected user-facing image data products from ``/repo/prompt`` via image service - (TBD)
+* Exposure of the PPDB tables from Postgres via TAP - April 2024 or as soon thereafter as the actual data are available
+
+Is this the same Postgres server and the same TAP service as the one serving ``/repo/embargo`` ObsCore?
```

Comment:
Is it safer to use a separate server and TAP service to access ``/repo/prompt`` and ``/repo/embargo`` so that we don't have to worry about security issues with the former?